pkg/api/vlabs/validate.go

```diff
@@ -469,8 +469,8 @@ func (a *Properties) validateAgentPoolProfiles(isUpdate bool) error {
 
 		// validate os type is linux if dual stack feature is enabled
 		if a.FeatureFlags.IsIPv6DualStackEnabled() || a.FeatureFlags.IsIPv6OnlyEnabled() {
-			if agentPoolProfile.OSType == Windows {
-				return errors.Errorf("Dual stack and single stack IPv6 feature is supported only with Linux, but agent pool '%s' is of os type %s", agentPoolProfile.Name, agentPoolProfile.OSType)
+			if agentPoolProfile.OSType == Windows && !common.IsKubernetesVersionGe(a.OrchestratorProfile.OrchestratorVersion, "1.19.0") {
```

Comment:
Should we also check the SKU here and error out if not `2004`?

Reply:
Also, have we tested single stack IPv6 on windows?

Reply:
> Should we also check the SKU here and error out if not `2004`?

I would like to do that, the challenge I see there is that the template could be using a custom image reference, which might not indicate the OS version.

> Also, have we tested single stack IPv6 on windows?

Nope, I haven't tested single stack IPv6, would there be anything unusual there? Do you suggest keeping it disabled for Windows?

Reply:
we have an open issue for adding tests: https://github.com/Azure/aks-engine/issues/3568

Reply:
I would recommend keeping `IPv6` only disabled until we validate for windows + azure CNI

Reply:
Updated.